nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -52,7 +52,7 @@ labels:
     max-ready-age: 600
   - name: iosxrv-7.0.2
     max-ready-age: 600
-  - name: nxos-7.0.3
```

Comment:
Can we not remove this entry, instead add a new entry with the new label? Same applies for all the changes.

Reply:
Done.